contracts/dynamic-callee-contract/tests/integration.rs

```diff
@@ -207,3 +208,26 @@ fn callable_point_pong_env_works() {
     let result: Env = from_slice(&serialized_return).unwrap();
     assert_eq!(result.contract.address, Addr::unchecked(MOCK_CONTRACT_ADDR));
 }
+
+#[test]
+fn callable_point_callee_panic_works() {
```

Comment:
I think `works` is a little ambiguous in this part. More descriptive (like `issues_runtime_error`) name is better.